orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccountInterventionService.java

```diff
@@ -125,6 +125,20 @@ private AccountInterventionStatus retrieveAccountStatus(String internalPairwiseS
         return accountInterventionStatus;
     }
 
+    public void doAccountIntervention(AccountInterventionStatus accountInterventionStatus) {
+        if (accountInterventionStatus.blocked()) {
+            LOG.info("Account is blocked");
+            // TODO: (ATO-171) back channel logout + (ATO-170) redirect to blocked page
+        } else if (accountInterventionStatus.suspended()
+                || accountInterventionStatus.resetPassword()
+                || accountInterventionStatus.reproveIdentity()) {
+            LOG.info(
+                    "Account is suspended, requires a password reset, or requires identity to be reproved");
+            // TODO: (ATO-171) back channel logout + (ATO-170) redirect to suspended
+            // page
+        }
+    }
```

Comment:
This shouldn't be in the shared service in this way. The logic of which flags result in which interventions depends on where in the journey it is called from. For example, suspended does not result in an intervention if on an ID journey when after auth.